room/compiler/src/main/kotlin/androidx/room/processor/CustomConverterProcessor.kt

```diff
@@ -70,7 +76,7 @@ class CustomConverterProcessor(val context: Context, val element: XTypeElement)
         }
     }
 
-    fun process(): List<CustomTypeConverter> {
+    fun process(addTypeConverterFactory: (XType) -> Unit = {}): List<CustomTypeConverter> {
```

Comment:
If we add a `private val` to `CustomConverterProcessor` in its constructor then we don't need this param since in `findConverters` can build a `CustomConverterProcessor` with the add lambda. This keeps `process()` clean as the processor has been built with everything it needs. It should also not have a default value.